server/api/jurisdictions.py

```diff
@@ -10,6 +10,8 @@
 from sqlalchemy.orm import joinedload
 from werkzeug.exceptions import Conflict, BadRequest
 
+from server.api.activity import serialize_activity
```

Comment:
Use relative imports for consistency

Reply:
Sorry, I need to update my IDE autocomplete config to not do this

Reply:
mine does it too lol. let me know if you figure out how to change it